lib/iris/coord_systems.py

```diff
@@ -820,3 +820,51 @@ def as_cartopy_crs(self):
 
     def as_cartopy_projection(self):
         return self.as_cartopy_crs()
+
+
+class Mercator(CoordSystem):
+    """
+    A coordinate system in the Mercator projection.
+
+    """
+
+    grid_mapping_name = "mercator"
+
+    def __init__(self, central_lon=0, ellipsoid=None):
+        """
+        Constructs a Mercator coord system.
+
+        Args:
+            * central_lon
+                    The central longitude.
+
+        Kwargs:
+
+            * ellipsoid
+                    :class:`GeogCS` defining the ellipsoid.
+
+        """
+
+        #: True longitude of planar origin in degrees.
+        self.central_lon = central_lon
+        #: Ellipsoid definition.
+        self.ellipsoid = ellipsoid
+
+    def __repr__(self):
+        return "Mercator(central_lon={!r}, "\
+               "ellipsoid={!r})".format(
```

Comment:
Alternatively (and a pattern I personally prefer):

```python
res = 'Mercator(central_longitude={!r}, ellipsoid={!r})'
return res.format(self.central_longitude, self.ellipsoid)
```

Saves the slightly forced splitting of lines. I'm also assuming you've changed the name of `central_longitude` :wink: